assets/prototype/domain/eventEditor/tickets/ticketsList/AddNewTicketButton.js

```diff
@@ -1,9 +1,12 @@
-import { useState } from '@wordpress/element';
+/**
+ * External dependencies
+ */
+import * as React from 'react';
```

Comment:
That could be better, no?
```suggestion
import React, { useState } from 'react';
```

Reply:
Yes, updated, thx.